docs/api/apiv3/components/schemas/file_link_model.yml

```diff
@@ -114,13 +137,163 @@ example:
     lastModifiedAt: '2021-12-20T14:00:13.987Z'
     createdByName: Luke Skywalker
     lastModifiedByName: Anakin Skywalker
+  _embedded:
+    storage:
+      id: 1337
+      _type: Storage
+      href: https://nextcloud.deathstar.rocks/
+      storageType: nextcloud
+      createdAt: '2021-12-20T13:37:00.211Z'
+      updatedAt: '2021-12-20T13:37:00.211Z'
+      _links:
+        self:
+          href: /api/v3/storages/1337
+          title: storage
+    container:
+      _type: WorkPackage
```

Comment:
Any chance we can `$ref` that?

Reply:
`$ref` references objects, our type is a string. not only in this model, but overall. Any good reason to work on changing that?

Reply:
referenced the single examples

Reply:
example references doesn't get resolved, so we use now shortened examples in the `_embedded` section